vector/src/main/java/im/vector/app/features/home/room/list/home/HomeRoomListFragment.kt

```diff
@@ -180,6 +183,14 @@ class HomeRoomListFragment @Inject constructor(
                     }
                 }.adapter
             }
+            is HomeRoomSection.RecentRoomsData -> RecentRoomCarouselController(
```

Comment:
I see `RecentRoomCarouselController` is injectable. I think we should inject this controller in this fragment instead of `AvatarRenderer`.